src/export-transactions.ts

```diff
@@ -87,9 +73,8 @@ async function getTopicTxs(network: number, startblock: number, topic: Topic) {
   return events
 }
 
-async function getTransactions(name: string, tokenAddress: string, network: number, address: string) {
-  let token = tokens[tokenAddress]
-  const url = `https://api.covalenthq.com/v1/${network}/address/${address}/transfers_v2/?key=${COVALENT_API_KEY}&contract-address=${tokenAddress}&page-size=500000`
+async function getTransactions(name: string, tokenAddress: string, network: Network, address: string) {
+  const url = `https://api.covalenthq.com/v1/${network.id}/address/${address}/transfers_v2/?key=${COVALENT_API_KEY}&contract-address=${tokenAddress}&page-size=500000`
```

Comment:
I know it's a high number, but eventually we are going to need to check that there are no more records in the next page, right?

If that's the case I'm ok with creating the ticket and merging as is.

Reply:
maybe use something like Governance's `inBatches`

Reply:
Glad you mentioned it, I've already included it in my next PR!

<img width="977" alt="image" src="https://user-images.githubusercontent.com/45410089/186443036-1a78ca38-5dac-4840-ad8a-ab771901c08f.png">
<img width="977" alt="image" src="https://user-images.githubusercontent.com/45410089/186443619-cb123db2-928f-4a88-af47-26ec84898d6b.png">